plugins/services/src/js/components/HighOrderServiceConfiguration.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import Pod from "../structs/Pod";
+import PodConfigurationContainer
+  from "../containers/pod-configuration/PodConfigurationContainer";
+import Service from "../structs/Service";
+import ServiceConfigurationContainer
+  from "../containers/service-configuration/ServiceConfigurationContainer";
+
+const HighOrderServiceConfiguration = props => {
```

Comment:
Please adjust the component name as it's not an good practice to name components after patterns. How about something like `GeneralServiceConfigurationContainer`. 

Side note: The naming of the `ServiceConfigurationContainer` seems off as pods are services as well...